src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,129 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private final Node falseHead;
```

Comment:
This field looks pretty unobvious. 
Let's better create two separate fields with names `head` and `tail`